config.tfvars

```diff
@@ -85,6 +85,9 @@ max_cluster_capacity = 5
 # List of addtional namespaces to be created in the cluster
 #additional_namespaces = ["extra_namespace"]
 
+# Path to a JSON file with EBS and RDS snapshot IDs
+#snapshots_json_file_path = "test/dcapt-snapshots.json"
```

Comment:
/nit we should be consistent with the space after the `#` comment symbol. I think leaving a space in is better for readability and removing the space is more convenient as I can just uncomment the line and don't delete the leading space.

I am leaning towards leaving a space